WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -12,6 +12,16 @@ final class NewOrderViewModel: ObservableObject {
 
     private var cancellables: Set<AnyCancellable> = []
 
+    /// Indicates whether user has made any changes
+    ///
+    var hasChanges: Bool {
+        productRows.isNotEmpty ||
+        selectedProductViewModel != nil ||
+        currentOrderStatus != .pending ||
+        paymentDataViewModel.hasChangesMade ||
+        customerDataViewModel.isDataAvailable
+    }
```

Comment:
We should be able to simplify this to check that the order isn't an empty new order:

```suggestion
    var hasChanges: Bool {
        orderSynchronizer.order != OrderFactory.emptyNewOrder
    }
```
(In which case we won't need the new `paymentDataViewModel.hasChangesMade` property.)

Reply:
Thanks for letting me know about this too! This is so much nicer 💯